DQM/TrackingMonitor/src/TrackBuildingAnalyzer.cc

```diff
@@ -296,9 +298,9 @@ void TrackBuildingAnalyzer::analyze
   GlobalPoint  v(v0.x()-bs.x0(),v0.y()-bs.y0(),v0.z()-bs.z0());
   
   double pt           = sqrt(state.globalMomentum().perp2());
-  double eta          = state.globalMomentum().eta();
-  double phi          = state.globalMomentum().phi();
-  double theta        = state.globalMomentum().theta();
+  double eta          = state.globalPosition().eta();
```

Comment:
while it's not in reco code, this looks odd to me.
I just wanted to check that these should be the detector surface eta/phi/theta values.
... same question for inner/outer position eta,phi (the innermost state position eta, say, for r_hit~4cm will be anywhere from ~-3 to +3  depending where the z_POCA is).